crates/tako/src/server/rpc.rs

```diff
@@ -168,6 +170,19 @@ async fn worker_rpc_loop(
     let snd_loop =
         forward_queue_to_sealed_sink(queue_receiver, connection.sender, connection.sealer);
 
+    let overview_event_loop = async move {
```

Comment:
This RPC loop runs for every worker, so like this you would periodically download overviews for all workers, inside each worker's RPC loop.